src/crosswordgenerator.ts

```diff
@@ -16,138 +17,176 @@ const emptyTile: tileCrossWord = {
 export async function generateCrossword(questions: question[]): Promise<tileCrossWord[][]>{
     let wordCount = questions.length;
     let answers: answer[] = [];
-    for(let i = 0; i < wordCount; i++){
+    let currentAnswers: answer[] = [];
+    questions.forEach((question,index) => {
         answers.push({
-            answer: questions[i].answer,
-            questNumber: i+1,
+            answer: question.answer,
+            questNumber: index+1,
         })
-    }
-
-    let crossword: tileCrossWord[][] = await simpleCrossWord(answers);
+        currentAnswers.push({
```

Comment:
Is If I got it right this is redundant with the loop in line 34 and can be done there by setting the initial score lower than a possible score